testsuite/forge.py

```diff
@@ -488,6 +488,58 @@ def get_humio_logs_link(
     )
 
 
+def get_axiom_link_for_test_runner_logs(
+    forge_namespace: str,
+    time_filter: Union[bool, Tuple[datetime, datetime]],
+) -> str:
+    """Get a link to the forge test runner logs in axiom for a given test run in a given namespace"""
+
+    apl_query = f"""
+        k8s
+        | where ['k8s.cluster'] contains "forge" and ['k8s.container_name'] != "calico-node" and ['k8s.namespace'] != "calico-apiserver" and ['k8s.container_name'] != "kube-proxy" and 
```

Comment:
can we break this up so its a little more readable?

```
constraints = [
  'cluster contains forge',
  'container_name != calico',
  'namespace != calico',
  ...
]
'k8s | where ' + ' and '.join(constraints)
```

Or can we keep the saved query server side?

Reply:
we can't keep it server side yet (they're working on it though).
In terms of breaking this up - I don't think this a good idea. If anything breaking this up makes it less readible as it's less-copy and pasteable into axiom's UI and vice versa.